packages/toolpad-app/src/appDom/index.ts

```diff
@@ -727,6 +727,16 @@ export function moveNode<Parent extends AppDomNode, Child extends AppDomNode>(
   return setNodeParent(dom, node, parent.id, parentProp, parentIndex);
 }
 
+export function duplicateNode<Child extends ElementNode>(dom: AppDom, node: Child) {
```

Comment:
Should this function also duplicate the children?

Reply:
Good catch! Thanks I didn't think about children, gonna adjust the code. I'm wondering is there any other component (or other situation) besides `Paper` that can have children?